Meta/CMake/freedesktop/org.ladybird.Ladybird.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.ladybird.Ladybird</id>
+
+  <name>Ladybird</name>
+  <summary>Truly independent web browser</summary>
+
+  <developer id="org.ladybird">
+    <name>Ladybird Browser</name>
+  </developer>
+
+  <url type="homepage">https://ladybird.org</url>
+  <url type="bugtracker">https://github.com/LadybirdBrowser/ladybird/issues</url>
+  <url type="donation">https://donorbox.org/ladybird</url>
+  <url type="vcs-browser">https://github.com/LadybirdBrowser/ladybird</url>
+  <url type="contact">https://discord.gg/nvfjVJ4Svh</url>
+
+  <metadata_license>0BSD</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+  <content_rating type="oars-1.1" />
+
```

Comment:
Not sure how this works, but looks like we could use https://freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-branding to set our black and purple colors from the branding guidelines (pinned comment in #website)